scripts/install_hooks.sh

```diff
@@ -10,7 +10,9 @@ if cat <<EOF > $ROOT_DIR/.git/hooks/pre-push
 
 cargo clippy --all-targets || { echo "Error: clippy did not pass - aborting push. Please run 'cargo clippy --all-targets'." ; exit 1 ; }
 
-cargo +nightly fmt -- --check --config unstable_features=true --config imports_granularity=Crate || { echo "Error: format check failed - aborting push. Please run 'cargo +nightly fmt'." ; exit 1 ; }
+ln -sf toolchains/nightly/rust-toolchain.toml
+cargo fmt -- --check --config unstable_features=true --config imports_granularity=Crate || { echo "Error: format check failed - aborting push. Please run 'cargo +nightly fmt'." ; exit 1 ; }
+ln -sf toolchains/stable/rust-toolchain.toml
```

Comment:
This will leave the user's project configured for nightly if `cargo fmt` fails — is this what we want?